rero_ils/permissions.py

```diff
@@ -50,12 +50,13 @@ def login_and_librarian():
 
 def can_access_item(user=None, item=None):
     """User has librarian role and logged in and in same item organisation."""
-    if not user:
-        user = current_user
     if item:
-        patron = Patron.get_patron_by_user(user)
-        if patron.organisation_pid == item.organisation_pid:
-            return user.is_authenticated and librarian_permission.can()
+        if not user:
+            user = current_user
+            if user.is_authenticated:
+                patron = Patron.get_patron_by_user(user)
+                if patron.organisation_pid == item.organisation_pid:
+                    return user.is_authenticated and librarian_permission.can()
```

Comment:
This test s already done in line 56:
`return librarian_permission.can()` shoud be enough.